app/views/kaminari/_first_page.html.erb

```diff
@@ -0,0 +1,3 @@
+<li class="page-item">
+  <%= link_to_unless current_page.first?, raw(t 'views.pagination.first'), url, remote: remote, class: 'page-link' %>
```

Comment:
what is the function of `raw`?

Reply:
Raw is a string container, which provides security in it using the HTML safe buffer, it is used because the data does not come from the user and there will be no collision by nil arrivals, unlike method html_safe.